locales/ca/home.json

```diff
@@ -0,0 +1,24 @@
+{
+   "heroSection": {
+      "button": "Començar",
+      "subtitle": "Genera llistes de música a Spotify basat en les cançons que estàn tocant en aquell tour, utilizant la API de Setlist.fm",
+      "title": "Generador de llistes de <0>Spotify</0> utilitzant una llista de grups musicals"
+   },
+   "howItWorksSection": {
+      "step1": {
+         "title": "Inicia sessió en el teu compte de Spotify"
+      },
+      "step2": {
+         "title": "Busca i selecciona els artistes o grups"
```

Comment:
I would say "artistes o grups" is a bit redundant. What about leaving "artists/artistes/artistas" in all languages?